containers/runner/Dockerfile

```diff
@@ -9,9 +9,14 @@ RUN dnf -y update && \
     git \
     virt-install \
     libguestfs-tools \
+    genisoimage \
     lorax-lmc-virt \
     parallel \
-    python3-libvirt && \
+    python3-libvirt \
+    createrepo \
+    python3-rpmfluff \
+    squid \
+    openssh-clients && \
```

Comment:
This is more than just genisoimage...?

Reply:
Right, that's the "Also add the other recently added dependencies.." part of the commit message. See PR #367 and PR #362

Reply:
I can split up the commits if you prefer, of course.

Reply:
Ah, sorry, I missed that :-/ That works, of course.